bee-metrics/examples/memory_usage.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_metrics::{metrics::MemoryUsage, serve_metrics, Registry};
+use tokio::time::Duration;
+
+async fn update_memory_usage(memory_usage: MemoryUsage) {
+    let mut vec = Vec::new();
+
+    loop {
+        // Add 10KiB of data every second and update the metric.
+        vec.extend_from_slice(&[0u8; 10240]);
+
+        memory_usage.update().await;
+
+        tokio::time::sleep(Duration::from_secs(1)).await;
+    }
```

Comment:
Can we use `tokio::time::interval` here? 
Let's also introduce the constant `MEMORY_UPDATE_INTERVAL` of type `Duration`?

Reply:
overlooked that it's just example code